grafana/provisioning/dashboards/chain.json

```diff
@@ -397,6 +397,11 @@
   "templating": {
     "list": [
       {
+        "current": {
+          "tags": [],
+          "text": "default",
+          "value": "default"
+        },
```

Comment:
Shouldn't this be set _below_ where the namespace label/name is actually defined?

Reply:
Not quite sure what you're referring to. The order of the filters is datasource, namespace, node which is a logical ordering. What specifically do you think should be moved lower?